ThirdParty.json

```diff
@@ -65,6 +65,12 @@
     "version": "0.2.10",
     "license": ["MIT"]
   },
+  {
+    "name": "libjpeg-turbo",
+    "url": "https://github.com/libjpeg-turbo/libjpeg-turbo",
+    "version": "2.1.91",
+    "license": ["BSD-3-Clause"]
```

Comment:
I think it's more complicated than this:
https://github.com/libjpeg-turbo/libjpeg-turbo/blob/main/LICENSE.md

If I'm reading this correctly, different parts of the code are covered by different licenses. So we should list all three: `IJG`, `BSD-3-Clause`, and `zlib`. Two of the three we're using already, and are definitely fine for us (BSD and zlib). The IJG license is an uncommon one, but seems to be BSD-ish and fine:
https://github.com/libjpeg-turbo/libjpeg-turbo/blob/main/README.ijg